compiler-rt/lib/sanitizer_common/sanitizer_common_nolibc.cpp

```diff
@@ -26,6 +26,7 @@ void LogMessageOnPrintf(const char *str) {}
 void WriteToSyslog(const char *buffer) {}
 void Abort() { internal__exit(1); }
 bool CreateDir(const char *pathname) { return false; }
+bool ShouldTreatRuntimeSecurely() { return false; }
```

Comment:
I think this is fine for now, but we should leave a TODO to revist whether this is the correct default. I can see arguments both ways "don't break anything" or "be more secure by default".